openQA.spec

```diff
@@ -53,6 +53,8 @@
 # all requirements needed by the tests, do not require on this in individual
 # sub-packages except for the devel package
 %define test_requires %common_requires %main_requires %python_scripts_requires %worker_requires
+%define         devel_requires %test_requires rsync ShellCheck os-autoinst-devel curl git postgresql-devel qemu qemu-kvm tar optipng postgresql-server xorg-x11-fonts rubygem(sass) sudo perl(Config) perl(Cwd) perl(Data::Dumper) perl(Digest::MD5) >= 2.55, perl(Data::OptList) perl(Devel::Cover::Report::Codecov) perl(LWP::Protocol::https) perl(SQL::SplitStatement) perl(XSLoader) >= 0.24
```

Comment:
there is a comma that looks suspicious - please branch

Reply:
branch? The comma is needed to separate every entry specifying a version comparision as already used in the macros defined above